inz-internetnz/Issues/votingcard/description.xml

```diff
@@ -0,0 +1,69 @@
+<IssueSpecification version="4">
+    <Name>
+        <en>Demo INZ Voting Card</en>
+        <nl>Demo INZ Voting Card</nl>
+    </Name>
+    <ShortName>
+        <en>InternetNZ Voting Card</en>
+        <nl>InternetNZ Voting Card</nl>
+    </ShortName>
+    <SchemeManager>irma-demo</SchemeManager>
+    <IssuerID>inz-internetnz</IssuerID>
+    <CredentialID>votingcard</CredentialID>
+    <Description>
+        <en>InternetNZ voting card</en>
+        <nl></nl>
+    </Description>
+    <ShouldBeSingleton>true</ShouldBeSingleton>
```

Comment:
I would argue that unless we have a very strict business rule, we could have more than one Voting Card (for different elections) at the same time in the wallet.

Maybe we could add an "id" attribute along with Election Title and use our PoC disclosing this credential not only with the type but also requiring that an [ID of a certain value](https://irma.app/docs/session-requests/#requesting-specific-attribute-values).

Reply:
I kinda wasn't sure about this myself as well. I agree, might be good to be able to issue different types of voting cards. We can use the `election` field to distinguish different elections.